nukkit/src/main/java/net/craftingstore/nukkit/events/DonationReceivedEvent.java

```diff
@@ -0,0 +1,93 @@
+package net.craftingstore.nukkit.events;
+
+import cn.nukkit.event.Cancellable;
+import cn.nukkit.event.Event;
+import cn.nukkit.event.HandlerList;
+import net.craftingstore.core.models.donation.Donation;
+
+import java.util.UUID;
+
+public class DonationReceivedEvent extends Event implements Cancellable {
+    @Deprecated
+    private String command, username, packageName;
```

Comment:
You don't need to include the deprecated event properties from the bukkit/spigot plugin. These are there to provide backward compatibility, but since this is a completely new platform, we only have to keep the new property (donation).